src/routes/__layout.svelte

```diff
@@ -43,10 +43,10 @@
 
 		<nav id="sidebar" slot="sidebarLeft" class="m-2">
 			<h5><a href={`${base}/`} on:click={() => (openLeft = false)}>Svelte-spectre</a></h5>
+			<br />
```

Comment:
Isn't it a bad practice to use line breaks? May be CSS is better...

Reply:
it's normal practice, but fixed in css